source/components/organisms/SummaryList/SummaryList.tsx

```diff
@@ -128,9 +129,30 @@ const SummaryList: React.FC<Props> = ({
   help,
   editable,
 }) => {
-  // For development: show an error popup when something is configured wrong.
-  const showNotification = useNotification();
+  const [isModified, setModified] = useState(false);
 
+  const sortAnswers = (listItems: SummaryListItem[]) => {
+    listItems.forEach((item) => {
+      if (ArrayType.includes(item.type)) {
+        const category = categories?.find(
+          (categoryItem) => item.category === categoryItem.category
+        );
+        const sortField = category?.sortField;
+        if (sortField && Array.isArray(answers[item.id])) {
+          const answerCopy = [...answers[item.id]].sort(
+            (valueA: Answer, valueB: Answer) => {
```

Comment:
I think the sorting logic itself is complex enough to warrant refactoring into a separate function. Something like:

```typescript
const answerCopy = [...answers[item.id]].sort((a, b) => compareAnswerBySortField(a, b, sortField));
```

Reply:
Fixed